src/ray/common/bundle_spec.cc

```diff
@@ -118,30 +118,57 @@ std::string FormatPlacementGroupResource(const std::string &original_resource_na
       original_resource_name, bundle_spec.PlacementGroupId(), bundle_spec.Index());
 }
 
-bool IsBundleIndex(const std::string &resource,
-                   const PlacementGroupID &group_id,
-                   const int bundle_index) {
-  return resource.find(kGroupKeyword + std::to_string(bundle_index) + "_" +
-                       group_id.Hex()) != std::string::npos;
-}
-
 std::string GetOriginalResourceName(const std::string &resource) {
```

Comment:
Can we split the PG parsing refactoring into a separate PR? This is also a bit of a scary change that would be nice to avoid.

Reply:
Would you like to merge this after 2.1 release? Btw, I'd like to keep `ParsePgFormattedResource` to avoid reporting pg resources to metrics if possible!